common/src/main/java/com/microsoft/identity/common/adal/internal/AuthenticationConstants.java

```diff
@@ -1247,6 +1257,50 @@ private AuthenticationConstants() {
         public static final String REMOVE_ACCOUNT_FROM_SHARED_DEVICE = "REMOVE_ACCOUNT_FROM_SHARED_DEVICE";
     }
 
+
+    public static final class BrokerContentProvider{
+
+        public static final String CONTENT_SCHEME = "content://";
+
+        /** Paths for all broker api's **/
+        public static final String AUTHORITY = "microsoft.identity.broker";
+
+        public static final String HELLO_PATH = "/hello";
+
+        public static final String ACQUIRE_TOKEN_INTERACTIVE_PATH = "/acquireTokenInteractive";
+
+        public static final String ACQUIRE_TOKEN_SILENT_PATH = "/acquireTokenSilent";
+
+        public static final String GET_ACCOUNTS_PATH = "/getAccounts";
+
+        public static final String REMOVE_ACCOUNTS_PATH = "/removeAccounts";
+
+        public static final String GET_CURRENT_ACCOUNT_SHARED_DEVICE_PATH = "/getCurrentAccountSharedDevice";
+
+        public static final String GET_DEVICE_MODE_PATH = "/getDeviceMode";
+
+        public static final String SIGN_OUT_FROM_SHARED_DEVICE_PATH = "/signOutFromSharedDevice";
+
+
+        /** Codes for URI **/
+        public static final int HELLO_URI_CODE = 1;
+
+        public static final int ACQUIRE_TOKEN_INTERACTIVE_CODE = 2;
+
+        public static final int ACQUIRE_TOKEN_SILENT_CODE = 3;
+
+        public static final int GET_ACCOUNTS_CODE = 4;
+
+        public static final int REMOVE_ACCOUNTS_CODE = 5;
+
+        public static final int GET_CURRENT_ACCOUNT_SHARED_DEVICE_CODE = 6;
+
+        public static final int GET_DEVICE_MODE_CODE = 7;
+
+        public static final int SIGN_OUT_FROM_SHARED_DEVICE_CODE = 8;
+
```

Comment:
Should these integers be in a separate container?

Reply:
These are related to `BrokerContentProvider` itself , so i'd prefer these to be within these class.